tsl/src/compression/batch_metadata_builder_minmax.c

```diff
@@ -0,0 +1,184 @@
+
```

Comment:
These files were just renamed, git doesn't detect it because there were also many renames inside.